neqo-transport/src/cc.rs

```diff
@@ -126,31 +128,40 @@ impl CongestionControl {
                 continue;
             }
 
-            if self.congestion_window < self.ssthresh {
-                self.congestion_window += pkt.size;
-                qinfo!([self], "slow start");
-                qlog::congestion_state_updated(
-                    &mut self.qlog,
-                    &mut self.qlog_curr_cong_state,
-                    CongestionState::SlowStart,
-                );
-            } else {
-                self.congestion_window += (MAX_DATAGRAM_SIZE * pkt.size) / self.congestion_window;
-                qinfo!([self], "congestion avoidance");
-                qlog::congestion_state_updated(
-                    &mut self.qlog,
-                    &mut self.qlog_curr_cong_state,
-                    CongestionState::CongestionAvoidance,
-                );
-            }
-            qlog::metrics_updated(
+            self.acked_bytes += pkt.size;
+        }
+        qtrace!([self], "ACK received, acked_bytes = {}", self.acked_bytes);
+
+        // Slow start, up to the slow start threshold.
+        if self.congestion_window < self.ssthresh {
+            let increase = min(self.ssthresh - self.congestion_window, self.acked_bytes);
+            self.congestion_window += increase;
+            self.acked_bytes -= increase;
```

Comment:
I think no need to update acked_bytes during slow start (it applies only to congestion avoidance)

Reply:
I'm using it as a scratch space/temporary variable in this case.  Also, if we overshoot ssthresh, any extra will stay around.